### PR TITLE
Created ionic action-list example

### PR DESCRIPTION
**Pull request by cmreddy071**
-  Created Ionic action-list local example

### CODE REVIEW

**Comment by cmreddy071** [comment]
Test cases modified.

File: src/app/app.component.scss [hunk 1]
```diff
@@ -0,0 +1,6 @@
+ion-item {
+    color: rgba(0,0,0,0.87);
```
Comment (joebochill) [inline src/app/app.component.scss]: this isn't a PX Blue color

File: src/app/app.component.scss [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+}
+.mat-icon {
+    cursor: pointer;
```
Comment (joebochill) [inline src/app/app.component.scss]: we probably do not want to have this rule for every mat-icon in the application - might make more sense to have it as an inline rule where needed. The cursor property also doesn't really have any effect in the context of a 'mobile' application.

File: src/global.scss [hunk 3]
```diff
@@ -13,14 +13,92 @@
...
+  height: 100%;
+  width: 100%;
+}
+
+ion-header, ion-toolbar {
+    color: map-get($pxb-white, 50);
+    --color : map-get($pxb-white, 50);
+    --background: map-get($pxb-blue, 500); 
+    background: map-get($pxb-blue, 500); 
+    font-size: 18px;
```
Comment (joebochill) [inline src/global.scss]: I thought we were using 16px in the header?

File: src/global.scss [hunk 4]
```diff
@@ -13,14 +13,92 @@
...
+  padding-left: unset;
+  padding-right: unset;
+  -webkit-padding-start: 0;
+  padding-inline-start: 0;
+  -webkit-padding-end: 0;
+  padding-inline-end: 0;
+}
+.action-sheet-wrapper.sc-ion-action-sheet-ios {  
+  bottom: -35px;
+}
```
Comment (joebochill) [inline src/global.scss]: This is a lot of styling for a simple application. How much of this is actually necessary? Why are we writing styling rules specific to iOS? Lots of values for margin and padding that aren't multiples of 5. We should not have to specify browser prefixes at all (especially for a mobile app).

**Pull request merged**